concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-70	70	froehlich	cheerful	1976	MERRY	11.241	6.583	6.417	6.417	6.25	6.5	6.333	6.417	2	2.25	2	2.455	2.125	2.217	2.17	5.727	4.5	5.25	4.833	5.087	5.042	5.064	6.083	6.083	6.25	6.417	6.083	6.333	6.208	5.917	5.583	5.25	5	5.75	5.125	5.438	2.417	2.5	2.667	2.917	2.458	2.792	2.625	6.688	6.5	1.708
+Gruehn-2008-200-71	71	frustriert	frustated			4.47	1.833	1.75	2.333	2.25	1.792	2.292	2.042	6.25	5.75	6	5.333	6	5.667	5.833	1.833	2.167	3.167	2.333	2	2.75	2.375	3.75	4.667	3.75	4.5	4.208	4.125	4.167	3	2.917	1.75	2.667	2.958	2.208	2.583	3.25	2.75	2.917	3	3	2.958	2.979	1.375	2.063	1.958
+Gruehn-2008-200-72	72	furchtlos	fearless			.8	5.083	5.083	5.25	5.25	5.083	5.25	5.167	2.917	3.25	3.833	4.417	3.083	4.125	3.604	5.167	5.25	4.727	5.5	5.208	5.13	5.17	3.917	5.417	5	4.5	4.667	4.75	4.708	3.333	4.5	5.417	4.333	3.917	4.875	4.396	2.333	2	2.167	2.167	2.167	2.167	2.167	5.125	4.813	1.75
+Gruehn-2008-200-73	73	furchtsam	frightened			23.6	2.5	2.5	3.167	2.583	2.5	2.875	2.688	5.417	5.667	6	3.727	5.542	4.913	5.234	2.083	2.167	3	2.75	2.125	2.875	2.5	3.75	5.083	4.364	5.667	4.417	5.043	4.723	3	2.917	2.667	3	2.958	2.833	2.896	3.583	3.5	4	3.417	3.542	3.708	3.625	2.313	2.75	1.917
+Gruehn-2008-200-74	74	gebildet	educated			20.458	6.583	6.167	6.5	6.333	6.375	6.417	6.396	3	2.917	3.083	3.75	2.958	3.417	3.188	6	5.833	5.417	5.5	5.917	5.458	5.688	3.083	5.091	4.25	4.417	4.043	4.333	4.191	5.583	5.583	5.167	5.417	5.583	5.292	5.438	3.083	3.167	3.5	3.333	3.125	3.417	3.271	6.5	6.188	1.833
+Gruehn-2008-200-75	75	geduldig	patient	3884	PATIENT	5.869	5.583	5.583	6.167	5.583	5.583	5.875	5.729	2.5	2.417	2.5	2	2.458	2.25	2.354	5.417	5.333	4.417	4.583	5.375	4.5	4.938	3.5	3.333	4	3.667	3.417	3.826	3.617	4.417	4.167	5.5	4.833	4.292	5.167	4.729	3.583	4	3.667	4	3.792	3.833	3.813	6.25	6.438	1.375
+Gruehn-2008-200-76	76	gehemmt	inhibited			1.535	2.364	2.083	2.667	2.583	2.217	2.625	2.426	5.5	5.833	4.667	4.167	5.667	4.417	5.042	2.583	1.667	2.182	2.583	2.125	2.391	2.255	3.25	3.333	4	3.917	3.292	3.957	3.617	2.083	2.75	2.333	2.583	2.417	2.458	2.438	3.417	3.083	3.083	3.333	3.25	3.208	3.229	2.063	2.313	1.792
+Gruehn-2008-200-77	77	gehorsam	obedient			1.161	2.75	2.75	5	4.5	2.75	4.75	3.75	5.833	4.833	3.75	4.333	5.333	4.042	4.688	3.333	2.833	5.25	3.417	3.083	4.333	3.708	4.167	5.5	4.333	4.167	4.833	4.25	4.542	3.333	2.833	3.583	3.083	3.083	3.333	3.208	3.5	3.417	3.667	3.167	3.458	3.417	3.438	3.313	3.063	1.167
+Gruehn-2008-200-78	78	geizig	stingy	1774	STINGY	831	1.667	1.917	2	1.667	1.792	1.833	1.813	5.417	6	4.917	4.364	5.708	4.652	5.191	4.583	4.583	3.833	3.583	4.583	3.708	4.146	4.364	6.083	4.25	4.5	5.261	4.375	4.809	2.167	2.333	1.917	1.833	2.25	1.875	2.063	3.083	2.917	3.833	3.5	3	3.667	3.333	2.313	1.688	1.333
+Gruehn-2008-200-79	79	gelangweilt	bored	905	BORING	3.126	2.25	2.75	2.75	2.333	2.5	2.542	2.521	2.75	3	2.667	2.333	2.87	2.5	2.681	2.917	2.5	2.917	2.917	2.708	2.917	2.813	4.167	5.917	3.667	3.917	5.042	3.792	4.417	2.583	2.833	2.083	2.917	2.708	2.5	2.604	3	2	1.917	2.583	2.5	2.25	2.375	1.813	1.938	1.75
```

Comment:
`905 BORING` is the the attribute causing the boredom, not the state of being bored. I'd unmap.